example_package/config.yml

```diff
@@ -1,87 +1,82 @@
-### Keys used by sio2:
+title: Przykładowy tytuł
 
-# Title of the package is defined in `title` key.
-title: Example task package
-# Title can be different for different languages.
-# They are defined in `title_<lang>` keys.
-# <lang> is a language code (for example 'pl' or 'en').
-# Available languages are defined in settings of `oioioi` instance.
-title_pl: Przykładowe zadanie
+sinol_task_id: abc
+
+memory_limit: 262144 # kB
+
+time_limit: 1000 # milliseconds
 
-# Number of points for each group can be defined in `scores` key.
-# If this key is not specified, then all groups have the same number of points.
-# (if number of groups doesn't divide 100, then the last groups will have the remaining points).
-# Group 0 always has zero points.
 scores:
-  1: 40
-  2: 60
+  1: 20
+  2: 40
+  3: 20
+  4: 10
+  5: 10
+
+
+# For more options see: https://github.com/sio2project/sinol-make/blob/main/example_package/config.yml
+# Or here are some basic ones.
+
+
+### Interactive tasks
+
+# Extra compilation arguments can be defined in `extra_compile_args` key.
+# Each language can have different extra arguments.
+# Additional files used in compilation can be defined in `extra_compilation_files` key.
+# They are copied to the directory where the source code is compiled.
+# All languages have the same additional files.
+
+# extra_compilation_args:
+#   cpp: ['abclib.cpp']
+
+# extra_compilation_files: ['abclib.cpp', 'abclib.h']
+
+
+### Time
 
-# Time limit for all tests is defined in `time_limit` key.
 # More precise time limit for each group or test can be defined in `time_limits` key.
 # The more precise time limit has higher priority (first group, then global time limit).
-# Time limit is in milliseconds.
-time_limit: 1000
-time_limits:
-  1: 2000
 
-# Memory limit for all tests is defined in `memory_limit` key.
+# time_limits:
+#   2: 2000
+#   5: 7000
+
+
+### Memory
+
 # More precise memory limits can be defined in `memory_limits` key.
 # Same as with time limits, the more precise memory limit has higher priority.
-# Memory limit is in kB
-memory_limit: 262144
-memory_limits:
-  3: 131072
-
-# Time and memory limits can be overriden for other languages in `override_limits` key.
-# If this key is not specified, then all languages have the same limits.
-override_limits:
-  py:
-    time_limit: 5000
-    memory_limit: 524288
 
-# Extra compilation arguments can be defined in `extra_compile_args` key.
-# Each language can have different extra arguments.
+# memory_limits:
+#   3: 131072
+#   4: 131072
 
-# extra_compilation_args:
-#   cpp: 'abclib.cpp'
 
-# The arguments can also be in an array:
+### Title
 
-# extra_compilation_args:
-#   cpp:
-#     - 'abclib.cpp'
-#     - 'abclib2.cpp'
+# If there are Polish characters, they should be written for better readability.
```

Comment:
Czy jest potrzeba wspominać o tym iż można dawać polskie znaki?

Reply:
Tak, bo szu głosi propagandę, że to zapycha pythona